### ISSUE TITLE
Implements {DataFrame,Series}.empty

### ISSUE THREAD

**Issue by wjsi** [mars-project/mars#1830]
Implements {DataFrame,Series}.empty

**Comment by BobinMathew** [comment]
Hi @wjsi where do you want to implement this code?
* I am a beginner, can I do it?

**Comment by qinxuye** [comment]
> Hi @wjsi where do you want to implement this code?
> 
> * I am a beginner, can I do it?

`Good first issue` means this feature is friendly enough for beginners.

Properties for Series can be added in https://github.com/mars-project/mars/blob/1b3801ac669d810e5513351f8b66f436f082fb05/mars/dataframe/core.py#L670-L758.

`DataFrameData` is in the same file.

Looking forward to seeing PRs from you! Thank you in advance.

**Comment by BobinMathew** [comment]
Do you mean to include this code:-
```diff
@property
def SeriesData(self):
    return self.SeriesData.empty

@property
def DataFrameData(self):
   return self.DataFrameData.empty
```

**Comment by qinxuye** [comment]
> Do you mean to include this code:-
> 
> ```diff
> @property
> def SeriesData(self):
>     return self.SeriesData.empty
> 
> @property
> def DataFrameData(self):
>    return self.DataFrameData.empty
> ```

No, I mean the property `empty` can be added to `BaseSeriesData`, I just gave the position that you can consider to add.

**Comment by BobinMathew** [comment]
Do you mean to include this code in BaseSeriesData class ?
```
DataFrame.empty
Series.empty
```

**Comment by qinxuye** [comment]
> Do you mean to include this code in BaseSeriesData class ?
> 
> ```
> DataFrame.empty
> Series.empty
> ```

I mean, `empty` is a property, in order to implement it, you can add it to `BaseDataFrameData` as well as `BaseSeriesData`.

**Comment by BobinMathew** [comment]
Is the solution to this issue similar to this issue?
[Implements {DataFrame,Series}.explode #1713](https://github.com/mars-project/mars/issues/1713)

Does the solution contain this much code?

**Comment by suchendra-h** [comment]
Can I do this?

Returns True if DataFrame/Series is empty?

**Comment by qinxuye** [comment]
> Can I do this?
> 
> Returns True if DataFrame/Series is empty?

Sure, you're welcome, just send a PR when you think it's ready, feel free to ask for help.

**Comment by wjsi** [comment]
Resolved in #2155
